tailwind.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
                 full: '100%',
             },
             strokeWidth: {
-                2: '2px',
+                3: '3px',
```

Comment:
please keep the `stroke-2` ;) 
```suggestion
                2: '2px',
                3: '3px',
```

Reply:
This is part of tailwind defaults: https://tailwindcss.com/docs/stroke-width

Reply:
It's defined in the viewBox units, I see. Then I'd move this part from `extend` to overrides.

Reply:
Ok. If it´s tested and works, lets keep it like this

Reply:
Ok, I've changed it to unitless value.